docker/dev/nginx.conf

```diff
@@ -0,0 +1,40 @@
+events {
+    worker_connections 1024;
+}
+
+http {
+    server {
+        listen 8536;
+        server_name 127.0.0.1;
+        #access_log  off;
+
+        # Upload limit for pictshare
+        client_max_body_size 50M;
+
+        location / {
+            proxy_pass http://lemmy:8536;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header Host $host;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+            # WebSocket support
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection "upgrade";
+        }
+
+        location /pictrs/ {
+            proxy_pass http://pictrs:8080/;
```

Comment:
it could be dangerous exposing parts of pictrs that aren't in the `/image` scope, because then someone could overwrite image aliases using the /import endpoint.